app/controllers/followships_controller.rb

```diff
@@ -0,0 +1,37 @@
+class FollowshipsController < ApplicationController
```

Comment:
Please rename your controller to something more meaningful .

Reply:
Suggest me any name , according to me controller name is meaningful.

Reply:
Use your creativity. Think of something more meaningful.  This is your learning.